index.bs

```diff
@@ -0,0 +1,221 @@
+<pre class=metadata>
+Title: Layout Instability API
+Status: ED
```

Comment:
Reminder: this status can be changed to CG-Draft when we move this to WICG.

Reply:
done.